variable/test/comparison_test.cpp

```diff
@@ -66,12 +66,6 @@ TEST(ComparisonTest, variances_test) {
   EXPECT_EQ(not_equal(a, b), true * units::one);
 }
 
-TEST(ComparisonTest, less_dtypes_test) {
```

Comment:
👍 Though this is the only test that demonstrates mixed input types.

Reply:
Note that these tests should actually be removed, since the functions in `variable/` (as well as `dataset/` and `python/`) are generated from a template by CMake since recently.